### PR TITLE
Update Community Roles

### PR DESCRIPTION
**Pull request by Mitzi-Laszlo**
Updated role descriptions to accurately describe current roles and responsibilities as well as include conclusions from https://github.com/solid/community/issues/10

### CODE REVIEW

**Comment by csarven** [comment]
@Mitzi-Laszlo If you have no objections, I can take on solid-spec as well.

**Comment by Mitzi-Laszlo** [comment]
Management of the Solid spec and architecture (which I understand is split over four repositories) is a key role that @timbl has recently spoken about. Perhaps we need a specific role for Solid specifications and architechture management? @michieldejong who previously worked on the Unhosted spec may also be a candidate. @timbl will need to make the final decision on allocating individuals to roles.

**Comment by RubenVerborgh** [comment]
I think it is important that the governance of the spec is in the hands of people who know the history and have been involved in its writing.

This holds for any repository BTW: whenever possible, the people in governance roles preferably have a longer history with the repository itself. In general, we probably want to see roles as “earned” through contributions rather than “assigned”.

But indeed, definitely for the spec, @timbl should be very closely involved.

**Comment by melvincarvalho** [comment]
> But indeed, definitely for the spec, @timbl should be very closely involved.

+1

I think the spec would benefit from a version number, with changes between versions subject to review.

File: community-roles.md [hunk 1]
```diff
@@ -4,32 +4,34 @@ and the responsibilities associated with each role, where applicable.
...
 
-## Repositories Manager
-The Repositories Manager controls Github, NPM, and other Solid organisation properties related to the storage, management, and distribution of official Solid projects. They are responsible for keeping these properties clean and well-organised on behalf of the Solid community. The Repositories Manager has admin rights of Solid tools must attend the weekly recurring community support meeting.
+## Repository Manager
+The Repository Manager is responisble for keeping the properties of their specific repository well-organised and communicating any changes clearly and publicly. The Respoitory Manager has admin rights of the repositosry they manage and is expected to attend the weekly recurring community support meeting.
```
Comment (michielbdejong) [inline community-roles.md]: typo 'Respoitory' and 'repositosry'

Comment (TallTed) [reply community-roles.md]: also `responisble` typo

**Comment by Mitzi-Laszlo** [comment]
> I think the spec would benefit from a version number, with changes between versions subject to review.

@melvincarvalho is this along the lines of what you had in mind? https://github.com/solid/solid-spec/blob/master/CHANGELOG.md

File: community-roles.md (hunk 1 above)
Comment (Mitzi-Laszlo) [reply community-roles.md]: Include: "Repository Managers are responsible for moderating conversations to find consensus and merging pull requests on their respective repositories."

**Comment by Mitzi-Laszlo** [comment]
Perhaps include the W3C Solid Community Chair(s)? https://www.w3.org/community/solid/

**Comment by Mitzi-Laszlo** [comment]
@justinwb mentioned an interest to being a core contributor too

**Comment by RubenVerborgh** [comment]
Count me in too, but what exactly is "core"? Seems not yet defined.

**Comment by justinwb** [comment]
> @justinwb mentioned an interest to being a core contributor too

Just to keep the record accurate here - the point I raised was that this role was defined but was removed in an update approximately 4-6 weeks ago. @RubenVerborgh, @kjetilk, @megoth, and I were attributed to it prior to its removal. The point I raised was to restore it (which @timbl supported). I've pushed a couple commits to restore it and also make the description a bit more informative than what it was before.

**Comment by Mitzi-Laszlo** [comment]
Here's a related pull request with a suggested route forward https://github.com/solid/information/pull/80

**Pull request merged**